.github/workflows/deploy.yml

```diff
@@ -0,0 +1,75 @@
+name: Website Deployment
+
+on:
+  push:
+    branches:
+      - main
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "16"
```

Comment:
Node 16 is already passed it's EOL. We, at least me, are using node 18, so let's go with that for now. (20 is the current LTS version)